Apps/Contoso.Forms.Puppet/Contoso.Forms.Puppet/ModulePages/AppCenterContentPage.xaml.cs

```diff
@@ -50,6 +52,10 @@ protected override async void OnAppearing()
             base.OnAppearing();
             LogLevelLabel.Text = LogLevelNames[AppCenter.LogLevel];
             AppCenterEnabledSwitchCell.On = await AppCenter.IsEnabledAsync();
+            if (Application.Current.Properties.ContainsKey(UserIdKey) && Application.Current.Properties[UserIdKey] is string id)
+            {
+                UserIdEntryCell.Text = id;
```

Comment:
Do you know why we don't have a text binding in Puppet?
It confused me at the first look why it isn't `UserId`.
Puppet
```
        <TableSection Title="Logging">
            <EntryCell Label="User Id" x:Name="UserIdEntryCell" HorizontalTextAlignment="End" Completed="UserIdCompleted"/>
        </TableSection>
```
Demo
```
        <TableSection Title="Logging">
            <EntryCell Label="User Id" x:Name="UserIdEntryCell" HorizontalTextAlignment="End" Text="{Binding UserId,Mode=TwoWay}" />
        </TableSection>
```

Reply:
🤷‍♂

Reply:
Probably because the app is a test app and we're pretty lax on proper design practices? 😆